### PR TITLE
ci(docker): add first u-root container v0.14.0

### LINKED ISSUE #260: containers for u-root do not have sshd
To get `--interactive` working, we need `sshd` installed in the container. So we have to make our containers anyway.

### PR DESCRIPTION
**Pull request by AtomicFS**
fixes #260 

When I added the container for the first time I was seeing some validation errors. So just to make sure it is actual syntax issue in `compose.yaml` and not some python error I have also added extra validation step. It is unnecessary, but does not slow anything down, so I figured why not.

### CODE REVIEW

File: tests/test_uroot.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+set -Eeuo pipefail
+
+# Clone repo
+git clone --branch "${VERIFICATION_TEST_UROOT_VERSION}" --depth 1 https://github.com/u-root/u-root.git
+cd u-root
+
+# Make
+go build
+./u-root \
+	-defaultsh gosh \
```
Comment (MDr164) [inline tests/test_uroot.sh:12]: Is this actually still needed? Not sure when they made gosh the default upstream.

File: tests/test_uroot.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	-o initramfs.cpio \
+	boot coreboot-app \
+	./cmds/core/* \
+	./cmds/boot/*
```
Comment (MDr164) [inline tests/test_uroot.sh:16]: Shouldn't this already be part of the `boot` preset?

File: tests/test_uroot.sh (hunk 1 above)
Comment (AtomicFS) [reply tests/test_uroot.sh:12]: I don't know, I just blindly copied stuff from CodingVoid.

File: tests/test_uroot.sh (hunk 2 above)
Comment (AtomicFS) [reply tests/test_uroot.sh:16]: Yeah, when I look at `u-root`'s `README` I see that `u-root core boot` should be enough.

**Pull request merged**